tests/cypress/tests/e2e/mlr/form.stepdef.js

```diff
@@ -23,6 +23,7 @@ When("I submit a new MLR program", () => {
   traverseRoutes(template.routes);
 
   //Submit the program
+  cy.wait(5000);
```

Comment:
Is this wait intentional & necessary?

Reply:
It was intentional, but I'm not sure if it's necessary. I had some failures on deployment because the `Submit` button was not rendering yet!